code/datums/monitor_statistics.dm

```diff
@@ -0,0 +1,15 @@
+//This datum holds statistic that are needed for the state calculation and are not stored anywhere else
+GLOBAL_DATUM_INIT(monitor_statistics, /datum/monitor_statistics, new)
+
+/datum/monitor_statistics
+	var/Ancient_Queen = 0
+	var/Elder_Queen = 0
+	var/Ancient_T3 = 0
+	var/Elder_T3 = 0
+	var/Ancient_T2 = 0
+	var/Elder_T2 = 0
+	var/list/Miniguns_in_use = new() 
+	var/list/SADAR_in_use = new() 
+	var/list/B18_in_use = new()
+	var/list/B17_in_use = new()
+	var/OB_available = 0
```

Comment:
why not have this as variables on the subsystem?

Reply:
No justification, you think it would be cleaner?
I'm think that a better monitor would need more and more variables, hence separating it now would be cleaner

Reply:
Theres no reason when you could just have it on the monitor when its global already

Reply:
Should make it a variable on SSmonitor then 
var/datum/monitor_statistics/monitor_statistics = new

Reply:
Done something like that